src/libCrypto/EthCrypto.cpp

```diff
@@ -171,6 +172,32 @@ bool VerifyEcdsaSecp256k1(const bytes& sRandomNumber,
   return result;
 }
 
+bool SignEcdsaSecp256k1(const bytes& digest, const bytes& privKey,
+                        bytes& signature) {
+  std::unique_ptr<secp256k1_context, decltype(&secp256k1_context_destroy)>
```

Comment:
Yes, because that  function actually uses a static context, which is only working because apparently it doesn't have any writable global state, but noone can guarantee that. 

In other words, I wanted to remove that single static variable, because it doesn't save anything, and create context when it is needed.